src/api/routes.py

```diff
@@ -200,13 +201,13 @@ def create_category(user_id):
 
 #Get all of user's categories
 @api.route('/users/<int:user_id>/categories', methods=['GET'])
-@jwt_required()  # Requires a valid JWT token
+# @jwt_required()  # Requires a valid JWT token
```

Comment:
This should be an authenticated endpoint, i.e. `@jwt_required` should be set on this function